extension/httpfs/httpfs-extension.cpp

```diff
@@ -58,6 +58,12 @@ static void LoadInternal(DatabaseInstance &instance) {
 
 void HTTPFsExtension::Load(DuckDB &db) {
 	LoadInternal(*db.instance);
+	Connection con(db);
```

Comment:
These lines of code will increase the size of the HTTPFS extension by 10MB~ - because this causes the extension to pull in the Query functionality which then links it against the entirety of DuckDB. Not really your fault but perhaps worth waiting until after the release to put this in so we can hopefully resolve the issue of inflated extensions before then.